File: app/src/main/java/com/sesi/chris/animangaquiz/data/model/UpdateResponse.java
package com.sesi.chris.animangaquiz.data.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class UpdateResponse {
    @SerializedName("estatus")
    @Expose
    public String estatus;
    @SerializedName("error")
    @Expose
    public String error;


}

File: app/src/main/java/com/sesi/chris/animangaquiz/data/model/UpdateResponse.kt
package com.sesi.chris.animangaquiz.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class UpdateResponse(
    @SerializedName("estatus") @Expose
    var estatus: String,
    @SerializedName("error")
    @Expose
    var error: String
)
